java/gradle/ice.gradle

```diff
@@ -82,7 +82,7 @@ spotless {
 }
 
 // Run spotlessApply before compiling Java code
-compileJava.dependsOn 'spotlessApply'
+// compileJava.dependsOn 'spotlessApply'
```

Comment:
Why is this commented out?

Reply:
We shouldn't be doing this. Otherwise we'll never know if the formatting is bad.

Reply:
then let's remove it.